src/Artifacts.jl

```diff
@@ -532,4 +532,14 @@ function era5_surface_data_1979_2024_path(; context = nothing)
     )
 end
 
+"""
+"""
+function find_modis_lai_fluxnet_sites(site_ID; context = nothing)
```

Comment:
Maybe rename to `get_modis_lai_fluxnet_data`, and don't forget to add a docstring :)